packages/patternfly-4/react-core/src/components/Tabs/Tabs.js

```diff
@@ -45,22 +43,33 @@ const defaultProps = {
   id: null,
   isFilled: false,
   isSecondary: false,
-  leftScrollAriaLabel: 'Scroll left',
-  rightScrollAriaLabel: 'Scroll Right',
-  'aria-label': null,
+  'aria-label-left-scroll': 'Scroll left',
+  'aria-label-right-scroll': 'Scroll Right',
```

Comment:
When I define `aria-label-left-scroll` as a prop on the Tabs component, I'm not seeing that string included on the button.

Reply:
Yep, fixed in latest commit. TabContent is still a bit whacky.